src/lib/snark_worker/rpc_failed_to_generate_snark.ml

```diff
@@ -12,9 +12,7 @@ module Master = struct
 
   module T = struct
     type query =
-      Mina_stdlib.Bounded_types.Wrapped_error.Stable.V1.t
-      * Selector.Spec.Stable.Latest.t
-      * Signature_lib.Public_key.Compressed.Stable.Latest.t
+      Error.t * (unit, unit, unit) Spec.Partitioned.Poly.Stable.Latest.t
```

Comment:
Why `Mina_stdlib.Bounded_types.Wrapped_error.Stable.V1.t` was replaced with `Error.t`?

Reply:
Use of `Vx` is not good, but let's use the `Wrapped_error`, even if it's `Error.t` underneath